test/nbft/nbft-dump.c

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+/**
+ * This file is part of libnvme.
+ * Copyright (c) 2023 Red Hat Inc.
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <inttypes.h>
+#include "libnvme.h"
+
+static void print_hex(unsigned char *buf, int len)
+{
+	int i;
+
+	for (i = 0; i < len; i++, buf++)
+		printf("%x", *buf);
+}
+
+static void print_nbft(struct nbft_info *table)
+{
+	unsigned int i, j;
+	struct nbft_info_hfi **hfi;
+	struct nbft_info_security **sec;
+	struct nbft_info_discovery **disc;
+	struct nbft_info_subsystem_ns **ssns;
+
+	printf("raw_nbft_size=%zd\n", table->raw_nbft_size);
+
+	printf("host.id=");
+	print_hex(table->host.id, 16);
```

Comment:
Just nitpicking... Instead of `16` you could use `NVME_UUID_LEN` (but since this is just test code it's not really necessary).